src/hyperpod_nemo_adapter/collections/parts/fsdp_strategy.py

```diff
@@ -97,13 +91,8 @@ def __init__(
         self.smp_config_dict = self._setup_smp_config(cfg)
         self.app_state = SageMakerAppState()
 
-        # Init from original PT-Lightning policy to avoid megatron specific initialization
+        # Init from original PT-Lightening policy to avoid Megatron specific initialization
```

Comment:
I think lightning is correct one.

https://github.com/Lightning-AI/pytorch-lightning

Reply:
updated